scrapy_spider_metadata/_metadata.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any, Dict, Type
+
+from scrapy import Spider
+
+from scrapy_spider_metadata._params import Args
+
+ATTR_NAME = "metadata"
+
+
+def get_metadata_for_spider(spider_cls: Type[Spider]) -> Dict[str, Any]:
```

Comment:
What about shortening it as `get_spider_metadata`?

Reply:
I wanted to also have `get_metadata_for_spider_name` but then decided it's not a good API. It's fine to rename this one now.